CONTRIBUTING.md

```diff
@@ -44,11 +44,14 @@ Your new GitHub app will need the following repository permissions & events:
 + Issues: Read & write
 + Metadata: Read-only
 + Pull requests: Read & write
++ Security events: Read-only
 
 **Organization Permissions**:
 + Members: Read-Only
 
 **Subscribe to Events**:
++ Security advisory
```

Comment:
Do we need this? Aren't we only listening to the `Code scanning alert` event?

Reply:
![image](https://user-images.githubusercontent.com/3264377/132449402-ef7a8a9e-7ad6-4805-b33e-444ca1e7e0aa.png)

The permission makes me think we need it. It explicitly mentions "code scanning alerts" - I can test what happens without the permission

Reply:
So that's Security events, which is the permission you added on line 47 above, which is correct. But we don't need to listen to security advisory, just need to listen to the code scanning event on the line below.